packages/react-graphql/src/async/tests/component.test.tsx

```diff
@@ -143,7 +143,7 @@ describe('createAsyncQueryComponent()', () => {
 
       const client = createMockApolloClient();
       const watchQuerySpy = jest.spyOn(client, 'watchQuery');
-      watchQuerySpy.mockImplementation(() => ({subscribe() {}} as any));
+      watchQuerySpy.mockImplementation(() => ({subscribe() {} } as any));
```

Comment:
This seems to be extra spacing which will cause the linter to fail. See below on like 205 for same thing.